Makefile

```diff
@@ -98,13 +98,6 @@ test-cpp-kokkos:
 	cmake --build ./BuildTests --target runner
 	cmake --build ./BuildTests --target test
 
-.PHONY: benchmark
-benchmark:
-	cmake --build BuildBench --target clean || true
-	rm -rf ./BuildBench/CMakeCache.txt ./BuildBench/compiler_info.txt ./BuildBench/run_gate_benchmark.sh
-	cmake $(LIGHTNING_CPP_DIR) -BBuildBench -DBUILD_EXAMPLES=ON -DCMAKE_BUILD_TYPE=Release -DENABLE_AVX=ON
-	cmake --build ./BuildBench
-
```

Comment:
Is this already gone from the master branch?

Reply:
Yes! It's gone after adding the google-benchmark suite. There is now `gbenchmark`.

Reply:
It is still there.

Reply:
It was probably supposed to be gone, but for some reason, both are there.
We are fixing it now.